src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px auto 40px;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+  display: flex;
+  width: 166px;
+  height: 36px;
+  line-height: 18px;
+}
+
+.name {
+  margin: 0 auto 4px;
+}
```

Comment:
remove class `name` and move this code to  `card__name`
you can add margins for bem elements, you can't add margins only for bem blocks